src/main/java/de/doubleslash/keeptime/view/SettingsController.java

```diff
@@ -18,9 +25,15 @@
 import javafx.scene.control.Label;
 import javafx.scene.layout.Region;
 import javafx.scene.paint.Color;
+import javafx.stage.Modality;
 import javafx.stage.Stage;
 
 public class SettingsController {
+   AboutController aboutController;
+   Stage aboutStage;
+
+   private static final String OS_NAME = "os.name";

```

Comment:
das hast du jetzt ja in einer eigenen OS Klasse - kann hier also raus

Reply:
stimmt